src/ontology/pato-edit.obo

```diff
@@ -20359,6 +20358,32 @@ is_a: PATO:0000383 ! female
 property_value: http://purl.org/dc/elements/1.1/creator http://orcid.org/0000-0001-5208-3432
 property_value: http://purl.org/dc/elements/1.1/date 2018-11-16T22:52:49Z xsd:dateTime
 
+[Term]
+id: PATO:0040021
+name: single
+def: "A quality inhering in a bearer by virtue of the bearer's being present as one entity." [https://orcid.org/0000-0003-3162-7490]
```

Comment:
i dont like this

Reply:
i am sorry nicole.

Reply:
what don't you like, @matentzn? this term, the def, both? Do you not want to add it?